curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-the-date-object-by-building-a-date-formatter/646adc0e1c347a017b89942c.md

```diff
@@ -0,0 +1,44 @@
+---
+id: 646adc0e1c347a017b89942c
+title: Step 3
+challengeType: 0
+dashedName: step-3
+---
+
+# --description--
+
+To link the `script.js` file to the HTML file, you should place the script tag after the body tag's closing tag.
```

Comment:
Let's remove this step since this is the fourth project in the curriculum. 

By this point, we are supplying the campers with all of the HTML and CSS.

Reply:
removed